modules/creating-manifest-file-customized-br-ex-bridge.adoc

```diff
@@ -0,0 +1,193 @@
+= Optional: Creating a manifest object that includes a customized `br-ex` bridge
+
+ifndef::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+As an alternative to using the `configure-ovs.sh` shell script to set a `br-ex` bridge on a bare-metal platform, create a `MachineConfig` object that includes a customized `br-ex` bridge network configuration.
+
+:FeatureName: Creating a `MachineConfig` object that includes a customized `br-ex` bridge
+include::snippets/technology-preview.adoc[]
+endif::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+
+ifdef::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+As an alternative to using the `configure-ovs.sh` shell script to set a `br-ex` bridge on a bare-metal platform, create a `NodeNetworkConfigurationPolicy` object that includes a customized `br-ex` bridge network configuration.
+
+:FeatureName: Creating a ``NodeNetworkConfigurationPolicy` object that includes a customized `br-ex` bridge
+include::snippets/technology-preview.adoc[]
+endif::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+
+Consider the following use cases for creating manifest object that includes a customized `br-ex` bridge:
+
+* You want to make postinstallation changes to the bridge, such as changing the Open vSwitch (OVS) or OVN-Kubernetes `br-ex` bridge network. The `configure-ovs.sh` shell script does not support making postinstallation changes to the bridge.
+* You want to deploy the bridge on a different interface than the interface available on a host or server IP address.
+* You want to make advanced configurations to the bridge that are not possible with the `configure-ovs.sh` shell script. Using the script for these configurations might result in the bridge failing to connect multiple network interfaces and facilitating data forwarding between the interfaces.
+
+ifndef::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+
+[NOTE]
+====
+If you require an environment with a single network interface controller (NIC) and default network settings, use the `configure-ovs.sh` shell script.
+====
+
+After you install {op-system-first} and the system reboots, the Machine Config Operator injects Ignition configuration files into each node in your cluster, so that each node received the `br-ex` bridge network configuration. The Operator prevents `br-ex` bridge configuration conflicts by sending a signal to the `configure-ovs.sh` shell script to not configure the `br-ex` bridge.
+endif::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+
+.Prerequisites
+
+
+.Procedure
+ifndef::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+. Create a `MachineConfig` manifest file and define a customized `br-ex` bridge network configuration similiar to the following example:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker <1>
+  name: 10-br-ex-worker
+spec:
```

Comment:
This appears to be an NNCP spec in a machine-config. That won't work (I wish we could use NNCPs as the day 1 interface, but alas). We need the NMState-specific data from this (the "interfaces" field and everything inside it) as a separate YAML field in the docs, which gets base64-encoded into the contents.source field of the machine-config.